airbyte-cdk/python/airbyte_cdk/sources/streams/concurrent/availability_strategy.py

```diff
@@ -67,6 +67,7 @@ def check_availability(self, logger: logging.Logger) -> StreamAvailability:
         """
 
 
+@deprecated("This class is experimental. Use at your own risk.", category=ExperimentalClassWarning)
```

Comment:
@brianjlai probably better to call out that it should not be used at all, if we're ripping out availability strategies over mid-long term?

Reply:
ah yeah i think i carried that over from a merge of serhii's work which deprecated availability strategy in concurrent. I'll update this to say do not use